internal/resources/sample_bucket_schema.go

```diff
@@ -0,0 +1,42 @@
+package resources
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+)
+
+func SampleBucketSchema() schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Computed: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"name":                       stringAttribute(required, requiresReplace),
+			"organization_id":            stringAttribute(required, requiresReplace),
+			"project_id":                 stringAttribute(required, requiresReplace),
+			"cluster_id":                 stringAttribute(required, requiresReplace),
+			"type":                       stringDefaultAttribute("couchbase", optional, computed, requiresReplace, useStateForUnknown),
+			"storage_backend":            stringAttribute(optional, computed, requiresReplace, useStateForUnknown),
+			"memory_allocation_in_mb":    int64DefaultAttribute(200, optional, computed),
```

Comment:
Is there any specific reason why we haven’t made this field ‘requiresReplace’?

Reply:
requiresReplace triggers force replacement if the particular field value changes